sphinx/texinputs/Makefile_t

```diff
@@ -14,10 +14,10 @@ ALLIMGS = $(wildcard *.png *.gif *.jpg *.jpeg)
 
 # Prefix for archive names
 ARCHIVEPRREFIX =
-# Additional LaTeX options (used via latexmkrc/latexmkjarc file)
-LATEXOPTS = --interaction=nonstopmode
+# Additional LaTeX options (used by latexmk via latexmkrc/latexmkjarc file)
+export LATEXOPTS =
```

Comment:
If user defined LATEXOPTS in environment, this will overwrite it. But it already overwrote it even without [export directive](https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html): 
> make exports a variable only if it is either defined in the environment initially or set on the command line

Thus it was already the case with 1.5.x and earlier that LATEXOPTS had to be used on the command line, for example

    make latexpdf LATEXOPTS="--interaction=nonstopmode"